### DISCUSSION TITLE
Help me to write a tag query

### DISCUSSION THREAD

**Discussion by strypeeeee** [jaegertracing/jaeger#5384] [Q&A] [answered]
How can I write a tag query to search for tag "http.request.payload"  that includes "15606" for example? I want to search for it from the tags search filed:
![image](https://github.com/jaegertracing/jaeger/assets/167959937/e07133fe-f8db-4e90-9fac-610c8d4cb8b6)

![image](https://github.com/jaegertracing/jaeger/assets/167959937/1d3214a1-ee0c-45df-959a-9ef01f59f731)

So far I can't make it work even if I try to search for exact same value like on the screenshot above. I have tried to escape, add double quotes and etc.

**Comment by yurishkuro** [comment, marked as answer]
Jaeger does not support substring searches, only exact matches

**Reply by strypeeeee** [reply]
Well this is sad as it's a common situation where I have JSON as value for tags and I need to check if json includes key or value. The use case is with Cassandra DB. A workaround would be to manually query Cassandra and via the returned trace ID to search it in the UI but this complicates things a lot. It would be great if Jaeger could build in this feature.
